src/Control/Lens/Traversal.hs

```diff
@@ -1329,7 +1329,7 @@ deepOf r l = failing l (r . deepOf r l)
 --
 -- 'confusing' exploits the 'Yoneda' lemma to merge their separate uses of 'fmap' into a single 'fmap'.
 -- and it further exploits an interesting property of the right Kan lift (or 'Curried') to left associate
--- all of the uses of '(<*>)' to make it possible to fuse together more fmaps.
+-- all of the uses of '(\<*\>)' to make it possible to fuse together more fmaps.
```

Comment:
```diff
+--
+-- * without parens '<*>'
+-- * with parens @('<*>')@
+-- * your '(\<*\>)'
+--
```
![screenshot from 2018-06-19 19-47-47](https://user-images.githubusercontent.com/51087/41611815-abf6f6c8-73f9-11e8-8f98-8b158b9611e1.png)

Reply:
Thank you